tests/Resources/ti.ui.shortcut.addontest.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+
+const should = require('./utilities/assertions');
+
+describe('Titanium.UI.Shortcut', () => {
+
+	it.android('Ti.UI.Shortcut', () => {
+		should(Ti.UI.Shortcut).not.be.undefined();
+	});
+
+	it.android('createShortcut', () => {
+		should(Ti.UI.createShortcutItem).not.be.undefined();
+		should(Ti.UI.createShortcut).be.a.Function();
+
+		// ONLY compatible with Android 7.1+, end test early.
+		if (Ti.Platform.osname === 'android') {
```

Comment:
FYI, You can now use `OS_ANDROID` as a simple guard and the app/sdk build fills it in with a boolean (like with Alloy):
```js
if (OS_ANDROID) {
  // android-specific code
}
```